AGENTS.md

```
@@ -332,14 +361,39 @@ shipped anyway. Two ways they slip through: an assertion that degenerates to a
+exit path including SIGINT.
+
 ```bash
-SCRATCH="$(mktemp -d)"; trap 'rm -rf "$SCRATCH"' EXIT
-cp internal/pkg/target.go "$SCRATCH/target.bak.go"
-# ... revert the fix in place, or neuter its condition ...
-go test -count=1 ./internal/pkg/ -run TestTheNewPin   # MUST fail, for the right reason
-cp "$SCRATCH/target.bak.go" internal/pkg/target.go
+go run ./scripts/mutate path/to/the.spec     # exits non-zero unless every mutation is KILLED
+```
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)